test/src/api/commands/web-element/testGetRect.js

```diff
@@ -133,4 +133,28 @@ describe('element().getRect() command', function () {
     const resultValue = await resultPromise.value;
     assert.deepStrictEqual(resultValue, {height: 34, width: 443, x: 356, y: 381.5});
   });
+
+  it('test .element().rect()', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/rect',
+      method: 'GET',
+      response: JSON.stringify({
+        value: {height: 34, width: 443, x: 356, y: 381.5}
+      })
+    }, true);
```

Comment:
Just one last thing, instead of having this test case at the bottom, can you move it to where the similar test cases are present (just below `test .element().getLocation()`)?

Reply:
done in  f53ad0b